release-repo-scripts/README.md

```diff
@@ -34,12 +35,12 @@ than any other expected in the repository.
 
 #### Example
 
-To bump `ign-rendering6` and create a new `ign-rendering7` release repositories,
-first fork the `ign-rendering6` in GitHub.
+To bump `gz-rendering6` and create a new `gz-rendering7` release repositories,
+first fork the `gz-rendering6-release` in GitHub and call the fork `gz-rendering7-release`.
 
 ```bash
-git clone https://github.com/gz-release/gz-rendering6-release
-cd gz-rendering6-release
+git clone https://github.com/gz-release/gz-rendering7-release
```

Comment:
Just checking: Should this example still use `gz-rendering6`? If the goal is to create `gz-rendering7-release` from `gz-rendering6-release` this `git clone` command will fail as it doesn't exist yet, right?

Reply:
The instructions say to first fork the repository. It is the fork we are cloning, so I think it's correct.